flasgger/utils.py

```diff
@@ -37,8 +39,6 @@ def get(self, filepath, root=None):
                 root = os.path.dirname(os.path.abspath(frame_info[1]))
             except Exception:
                 root = None
-        else:
-            root = os.path.dirname(root)
```

Comment:
removed because this takes to one directory level up which we don't want. we want to be where we are